modules/protocol/inspircd.c

```diff
@@ -385,6 +385,12 @@ inspircd_quit_sts(struct user *u, const char *reason)
 static void
 inspircd_wallops_sts(const char *text)
 {
+	if (!me.bursting)
+	{
+		slog(LG_DEBUG, "inspircd_wallops_sts(): refusing to send SNONOTICE while bursting: %s", text);
```

Comment:
You should not log here, this could trigger an infinite loop if debug is sent to notices.

Reply:
I think there's a check in `vslog_ext` nowadays that should make it no-op when called recursively, so this is probably fine (but the log message might get dropped if we got here by trying to log something else)

Reply:
(that said, is this check supposed to be negated?)

Reply:
This should be resolved now.